### DISCUSSION TITLE
最具潜力的Frp，即将成为圈内传奇！【风林水起Frp】 - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by Muska-Ami** [Muska-Ami/muska-ami-blog#62] [General]
https://blog.1l1.icu/2023/07/29/zui-ju-qian-li-de-frp-ji-jiang-cheng-wei-quan-nei-chuan-qi-feng-lin-shui-qi-frp/ 

 视频请直接看 @云墨安 的视频：    你说的对，但是《风林水起Frp》是由 FLSQ团队 自主研发的一款全新 Frp。游戏发生在一个被称作「 互联网 」的幻想世界，在这里，被 15元一个月 选中的人将被授予「 100Mbps带宽 」，导引 流量 之力。你将扮演一位名为「 用户 」的神秘角色，在自由的旅行中邂逅性格各异、能力独特的 FLSQ团队成员 们，和他们 共同使用风林水起Frp ——同时，逐

**Comment by Lzasn** [comment]
好好好，真有你的👍

**Reply by llll415** [reply]
猎奇()
